catalog/core/catalog-core-metacardgroomerplugin/src/test/java/ddf/catalog/plugin/groomer/TestMetacardGroomerPlugin.java

```diff
@@ -63,23 +69,21 @@
  * @author ddf.isgs@lmco.com
  */
 public class TestMetacardGroomerPlugin {
```

Comment:
Good catch. There was one for `testUpdateStandard` but it only verified the `metacard.created` date was unchanged on an update. Added additional unit tests for `update` operations, including verification that empty `metacard.created` is set to current date/time.